client/render/ImageLocator.h

```diff
@@ -33,6 +33,7 @@ struct ImageLocator
 	bool verticalFlip = false;
 	bool horizontalFlip = false;
 	int8_t scalingFactor = 0; // 0 = auto / use default scaling
+	int8_t preScaledFactor = 1;
```

Comment:
Why `scalingFactor` is not enough? How does `preScaledFactor` differs from it?

Reply:
Tried to unify them but failed... to complicated...

`scalingFactor` - describes the desired factor to scale
`preScaledFactor` - describes the factor a image is already scaled before loading (e.g. there is no auto like in `preScaledFactor`)